src/mlpack/core/data/load.hpp

```diff
@@ -65,6 +65,12 @@ bool Load(const std::string& filename,
           const bool fatal = false,
           const bool transpose = true);
 
+template<typename eT>
+bool Load(const std::string& filename,
+          arma::SpMat<eT>& matrix,
+          const bool fatal = false,
+          const bool transpose = true);
```

Comment:
Can you add comprehensive documentation to this? This is a user-facing function, so it's really important that it's documented very clearly. :+1:

Reply:
Awesome, this looks great.  Thank you! :+1: